tests/inputs/wf_theiaprok_illumina_pe.json

```diff
@@ -4,6 +4,7 @@
     "theiaprok_illumina_pe.read2_raw": "bactopia-tests/data/species/portiera/illumina/SRR2838702_R2.fastq.gz",
     "theiaprok_illumina_pe.skip_screen": true,
     "theiaprok_illumina_pe.read_QC_trim.call_midas": false,
+    "theiaprok_illumina_pe.read_QC_trim.call_fastp": false,

```

Comment:
Similar to `assembler`, I'm wondering if it may make more sense to set a `read_processing` var to trigger either `trimmomatic` or `fastp`. This may be applicable to `bakta` v. `prokka`, too.

Reply:
^^ Just a thought for now and won't stop me from approving, but we should harmonize on the options logic before next version release

Reply:
I agree that that would be a useful change! Modifying the input now